internal/model/tabs/jobhisttab/jobhisttab.go

```diff
@@ -46,9 +50,57 @@ type Breakdowns struct {
 	JobPerPart generic.CountItemSlice
 }
 
+func NewUserInputs(t uint, starttime string, endtime string) generic.UserInputs {
+	var tmp_s string
+	var tmp_t textinput.Model
+
+	// User Input (for modifying the table/view)
+	userinput := generic.UserInputs {
+		FocusIndex: 0,
+		ParamTexts: make([]string, 3),
+		Params: make([]textinput.Model, 3),
+	}
+
+	// TODO: we should probably think of encapsulating this
```

Comment:
Why not just do a package struct var of type generic.UserInputs in this package, then from scom.go just fill out these three fields that need changing and initiate the model with that?

Reply:
My idea was that UserInputs should be _generic_ and used for defining a UI for any number of textinputs, really we could use this for the filter as well?

Reply:
Ah ok, understand now. Can be, needs a bit of thoughts and love before but yes, i agree with the idea. :+1: